### ISSUE TITLE
Context menu issue regarding text fields on "addons" page

### ISSUE THREAD

**Issue by Chikorita-Lover** [TurboWarp/scratch-gui#618]
### Expected Behavior

On the "addons" page for configuring TurboWarp addons, several addons use a color picker to set colors (such as the "Customizable Block Colors" addon). The text field in the color picker does not support the standard context menu (including actions such as copy and paste). This makes translating settings from Scratch Addons to TurboWarp more difficult as text cannot be copied or pasted.

### Actual Behavior

The text field in the color picker should support the standard context menu, and allow for the copying and pasting of text.

### Steps to Reproduce

1. Click "Addons" in the toolbar
2. Find an addon that uses a color field
3. Open the color picker
4. Attempt to right-click or open the context menu while in the color picker's text field
5. Observe results

### Operating System and Browser

Windows 10
Chrome 105.0

**Comment by GarboMuffin** [comment]
Yeah, the color picker built in to most browsers tends to not be very good. We should be using a custom picker instead such as the paint editor's color picker

**Comment by GarboMuffin** [comment]
Also, if you export settings from Scratch Addons, you should be able to import that JSON file into TurboWarp's addons page and every supported addon should just work

**Comment by Chikorita-Lover** [comment]
Yeah, exporting from Scratch Addons works for most addons, although it doesn't work for Customizable Block Colors.

**Comment by GarboMuffin** [comment]
It does work for that addon

**Comment by Chikorita-Lover** [comment]
It does? Weird, I must be doing something wrong

**Comment by GarboMuffin** [comment]
You should upload the JSON settings output by Scratch Addons that didn't import into TurboWarp properly in a GitHub comment or to contact@turbowarp.org if GitHub doesn't let you attach JSON files

**Comment by GarboMuffin** [comment]
Importing settings for that addon will be fixed in the next update

**Comment by GarboMuffin** [comment]
See if it works now

**Comment by Chikorita-Lover** [comment]
It works great—thanks for looking into it!
